src/develop/masks.h

```diff
@@ -327,7 +328,9 @@ void dt_masks_gui_form_remove(dt_masks_form_t *form, dt_masks_form_gui_t *gui, i
 void dt_masks_gui_form_test_create(dt_masks_form_t *form, dt_masks_form_gui_t *gui);
 void dt_masks_gui_form_save_creation(dt_develop_t *dev, struct dt_iop_module_t *module, dt_masks_form_t *form,
                                      dt_masks_form_gui_t *gui);
+dt_masks_form_t *dt_masks_group_create(dt_iop_module_t *module, dt_masks_type_t type);
```

Comment:
As this one is only used in body it should be removed from there as not a routine useful outside. And...